### DISCUSSION TITLE
Pin assignments in User_Setup.h not being used

### DISCUSSION THREAD

**Discussion by Bwanna** [Bodmer/TFT_eSPI#2046] [New processor/display support or capability request]
Hi,
I've created the following pin assignments in the User_Setup.h file:
**#define TFT_MOSI 23
#define TFT_MISO 19
#define TFT_SCLK 18
#define TFT_CS   15
_#define TFT_DC    2
#define TFT_RST   4_** 

However, when I run Read_User_Setup sketch I get the following:
**MOSI    = GPIO 23
MISO    = GPIO 19
SCK     = GPIO 18
TFT_CS   = GPIO 15
_TFT_DC   = GPIO 0
TFT_RST  = GPIO 2_**

Maybe I'm not doing something correctly, but assistance would be greatly appreciated. I also did try and include these defines in the sketch itself without any luck.

Thanks,

**Comment by Bodmer** [comment]
The pin choice suggests you are using an ESP32.

Look at the User_Setup_Select.h file to make sure no other setup is loaded other than  User_Setup.h.
Look at User_Setup.h to make sure each pin is only defined once in the whole file.

Other than that I have insufficient information to judge the cause.
